client/mass/about.ts

```diff
@@ -135,9 +135,17 @@ export class MassAbout {
 					.style('margin-left', '0px')
 					.on('click', async () => {
 						const state = app.getState()
-						const clearOnChange = state.clearOnChange
+						const clearOnChange = state.termdbConfig.selectCohort.clearOnChange
```

Comment:
purpose and structure of `clearOnChange` is not clear in dataset.ts

Reply:
I will add comment in a next PR. This allows to specify if the dataset should delete the plots or the filter when changing cohorts. So far it is only done for the profile, Edgar suggested to add this configuration to the dataset so any dataset with this config would behave the same.